lib/slacked/slack_post.rb

```diff
@@ -1,15 +1,19 @@
 module Slacked
   SLACK_PROFILE_IMAGE=':robot_face:'
-  SLACK_WEBHOOK_URL_KEY='SLACK_WEBHOOK'
+  SLACK_WEBHOOK_URL_KEY = 'SLACK_WEBHOOK'
   SLACK_DEFAULT_MESSAGE_KEY='SLACK_DEFAULT_MESSAGE'
-  SLACK_DEFAULT_CONFIG= {
-      icon_emoji: SLACK_PROFILE_IMAGE
-  }
+  SLACK_DEFAULT_CONFIG = {
+    icon_emoji: SLACK_PROFILE_IMAGE,
+    webhook_url: ENV[SLACK_WEBHOOK_URL_KEY]
+  }.freeze
 
   class << self
     def post message = ENV[SLACK_DEFAULT_MESSAGE_KEY], config = SLACK_DEFAULT_CONFIG
       return false if message.nil? || message.empty? || disabled?
-      slack_notifier.ping message, SLACK_DEFAULT_CONFIG.merge(config)
+
+      merged_configs = SLACK_DEFAULT_CONFIG.merge(config)
+      slack_notifier(merged_configs.fetch(:webhook_url, SLACK_WEBHOOK_URL_KEY))
```

Comment:
please move `merged_configs.fetch(:webhook_url, SLACK_WEBHOOK_URL_KEY)` to a variable, this way it is easier to understand this method call.

Reply:
Fair comment. That has been resolved.